### ISSUE TITLE
formatted_body examples are over-escaped

### ISSUE THREAD

**Issue by t3chguy** [matrix-org/matrix-spec#983]
**Link to problem area**:
https://spec.matrix.org/v1.2/client-server-api/#memote

**Issue**
What is wrong? What can we do to improve?
![image](https://user-images.githubusercontent.com/2403652/156788242-fcbb51d0-19c3-457e-a0b8-d271632c4c78.png)

these unicode escape sequences should just be lt/gt presumably

**Comment by zecakeh** [comment]
I don't think this happens anymore, e.g. the [`m.emote`](https://spec.matrix.org/v1.13/client-server-api/#memote) example now looks fine.
